app/src/main/java/com/parasoft/demoapp/util/ImageUtil.java

```diff
@@ -17,26 +22,52 @@
         PDAService pdaService = new PDAService();
+        if (viewHolder != null) {
+            editItemLayout(120, View.GONE, imageView, viewHolder);
+        }
         pdaService.getClient(ApiInterface.class).getImage(orderImage)
-            .enqueue(new Callback<ResponseBody>() {
-                @Override
-                public void onResponse(@NonNull Call<ResponseBody> call, @NonNull Response<ResponseBody> response) {
-                    if (response.code() == 200) {
-                        assert response.body() != null;
-                        Bitmap bitmap = BitmapFactory.decodeStream(response.body().byteStream());
-                        imageView.setImageBitmap(bitmap);
-                    } else {
-                        // TODO: Set a default local image when load image failed
+                .enqueue(new Callback<ResponseBody>() {
+                    @Override
+                    public void onResponse(@NonNull Call<ResponseBody> call, @NonNull Response<ResponseBody> response) {
+                        if (response.code() == 200) {
+                            if (viewHolder != null) {
+                                editItemLayout(5, View.VISIBLE, imageView, viewHolder);
+                            }
+                            assert response.body() != null;
+                            Bitmap bitmap = BitmapFactory.decodeStream(response.body().byteStream());
+                            imageView.setImageBitmap(bitmap);
+                        } else {
+                            showLoadingFailedImage(imageView, viewHolder);
+                        }
                     }
-                }
-
-                @Override
-                public void onFailure(@NonNull Call<ResponseBody> call, @NonNull Throwable t) {
-                    Log.e("OrderDialog", "Load image " + orderImage + " failed", t);
-                    // TODO: Set a default local image when load image failed
-                }
-            });
+
+                    @Override
+                    public void onFailure(@NonNull Call<ResponseBody> call, @NonNull Throwable t) {
+                        showLoadingFailedImage(imageView, viewHolder);
+                        Log.e("OrderDialog", "Load image " + orderImage + " failed", t);
+                    }
+                });
+    }
+
+    private static void showLoadingFailedImage (ImageView imageView, @Nullable OrderItemAdapter.ViewHolder viewHolder) {
+        imageView.getLayoutParams().height = dp2px(100);
+        imageView.getLayoutParams().width = dp2px(65);
+        imageView.setImageResource(R.mipmap.ic_image_loading_failed);
+        if (viewHolder != null) {
+            editItemLayout(170, View.GONE, imageView, viewHolder);
+        }
+    }
+
+    private static void editItemLayout (int leftMargin, int visibility, ImageView imageView, @Nullable OrderItemAdapter.ViewHolder viewHolder) {
```

Comment:
There's no need to change parameters like width, height, margin that frequent. We can simple set a fixed wrapper for image and replace the source inside.